src/swap/components/Swap.tsx

```diff
@@ -48,6 +49,7 @@ export function Swap({
       onError={onError}
       onStatus={onStatus}
       onSuccess={onSuccess}
+      isSponsored={isSponsored}
```

Comment:
nit: alphabetic imports

Reply:
done - this was from when we called it `sponsored`